src/simplicity.c

```diff
@@ -66,10 +67,22 @@ static void prv_main_window_load(Window *window) {
   s_shifting_layer = layer_create(full_bounds);
   layer_add_child(s_window_layer, s_shifting_layer);
 
+  #if defined PBL_PLATFORM_EMERY
```

Comment:
There are a lot of hardcoded numbers here and use of the platform defines...is this what we want to set as best practice for handling different screen sizes and resolutions?